### ISSUE TITLE
undefined is not an object (evaluating 'd.propTypes.style')

### ISSUE THREAD

**Issue by Unforgiven-wanda** [shoutem/ui#331]
Hello,
In a vanilla React-native project (Pepperoni app template), when running in debug this UI runs fine. However once I bundle my apk the app crashes at startup with the following error:
```
undefined is not an object (evaluating 'd.propTypes.style')
```
after some search I tried [this solution](https://github.com/facebook/react-native/issues/5655) with no luck however.

Has anyone else encountered a similar issue?
Thank you.

**Comment by mcxsic** [comment]
Same issue here :/ I'm trying to figure out what's going on...

**Comment by zrubenst** [comment]
+1

**Comment by erhutime** [comment]
+1

**Comment by santhosh77h** [comment]
+1 I'm facing this error only in release apk and app crashes but not using shoutem instead using native-base which has dependency of native-base-shoutem-theme I think this may causing the issue

**Comment by csoutham** [comment]
+1 Must admit I tried for several hours last night to get past this last night to give it a try but with no luck.

**Comment by robisaks** [comment]
I had the same problem, it ended up being an outdated module I was including.

https://github.com/d-a-n/react-native-modal-picker

I switched to an updated fork, and it worked

https://github.com/peacechen/react-native-modal-selector

**Comment by Vladimir-Vdovic** [comment]
All `PropTypes` issues have been resolved as of [version 0.22.0](https://www.npmjs.com/package/@shoutem/ui). Closing issue.
